apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -46,6 +47,37 @@ class PopUpMenu extends Component {
     }).isRequired,
     children: PropTypes.any,
     className: PropTypes.string,
+    isOpen: PropTypes.bool,
+    beforeClose: PropTypes.func,
+  };
+
+  render() {
+    return (
+      <Portal
+        closeOnEsc
+        closeOnOutsideClick
+        isOpened={this.props.isOpen}
+        beforeClose={this.props.beforeClose}
+      >
+        <RadiumMenuBubble
+          targetPoint={this.props.targetPoint}
+          className={this.props.className}
+          children={this.props.children}
+        />
+      </Portal>
+    );
+  }
+}
+export default PopUpMenu;
+
+class MenuBubble extends Component {
```

Comment:
It looks like the only version of this that is ever used is the one wrapped with `Radium`. So why not just drop the non-Radium one? i.e.

```jsx
export const MenuBubble = Radium(class MenuBubble extends Component {
  ...
});
```